content/it/docs/concepts/cluster-administration/cluster-administration-overview.md

```diff
@@ -12,7 +12,7 @@ Presuppone una certa dimestichezza con i core Kubernetes [concetti](/docs/concep
 {{% capture body %}}
 ## Planning a cluster
 
-Consulta le guide in [Scelta della soluzione giusta](/docs/setup/pick-right-solution/) per esempio su come pianificare, 
+Consulta le guide in [Setup](/docs/setup) per esempio su come pianificare, 
```

Comment:
@micheleberardi, is `Setup` correct or does it translate to something else in Italian?

Reply:
impostazioni

Reply:
So, do i need to change 'Setup' to  'impostazioni' ? or that's fine!

Reply:
@jnardiello thanks for offering your help as well (in https://github.com/kubernetes/community/pull/3745).

What do you think `Setup` should translate to (non-italian speaker here :p) ?

Reply:
I said "impostazioni" and the rest of the sentence seems fine, but it's better that @jnardiello give the final ok

Reply:
I would use "Setup" in this context (it's commonly used and perfectly acceptable in italian), "Impostazioni" could be mistaken as "Settings" (that's the literal translation).

The issue in this sentence is another one: `esempio` is singular, we need to use `esempi` (plural). A correct overall sentence is:
`Consulta le guide di Setup per avere degli esempi su come pianificare, impostare e configurare cluster Kubernetes.`